docker/production.yml

```diff
@@ -227,6 +227,20 @@ services:
       # Disable Elasticsearch routing via Traefik in production (we enable it in development)
       - 'traefik.enable=false'
 
+  portainer:
+    image: portainer/portainer:alpine
```

Comment:
What did you understand about using `portainer` vs. `portainer-ce`? https://hub.docker.com/r/portainer/portainer-ce.  I got the sense that the "open" version was moving to the Community Edition.

Reply:
oh, I should probably change this then.